app/services/imports/import_service.rb

```diff
@@ -13,6 +13,8 @@ def import_from(folder, create_method)
         file_io = @storage_service.get_file_io(filename)
         xml_document = Nokogiri::XML(file_io)
         send(create_method, xml_document)
+      rescue StandardError => e
```

Comment:
Not a blocker for this PR but I wonder if we'd be better off wrapping the import task in a transaction and just raising an error rather than logging (so the whole task either completes successfully or completely rolls back).

Reply:
Add the filename is the error log